driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionRangeDefaultExplicitEncryptionTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+
+package com.mongodb.client;
+
+import com.mongodb.ClientEncryptionSettings;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoNamespace;
+import com.mongodb.client.model.vault.EncryptOptions;
+import com.mongodb.client.model.vault.RangeOptions;
+import com.mongodb.client.vault.ClientEncryption;
+import com.mongodb.fixture.EncryptionFixture.KmsProviderType;
+import org.bson.BsonBinary;
+import org.bson.BsonInt32;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static com.mongodb.ClusterFixture.isServerlessTest;
+import static com.mongodb.ClusterFixture.isStandalone;
+import static com.mongodb.ClusterFixture.serverVersionAtLeast;
+import static com.mongodb.assertions.Assertions.assertTrue;
```

Comment:
We should use `org.junit.jupiter.api.Assertions.assertTrue` instead for checks in tests.

P.S. In the rare occasion when we need an assertion about the testing code itself, we should indeed use `com.mongodb.assertions.Assertions`, but here we don't have such a situation.